cumulus_library/template_sql/core_medication.sql.jinja

```diff
@@ -155,11 +151,7 @@ CREATE TABLE core__medication AS (
             t.r.code AS code,
             mmn.code.text AS display,
             t.r.system AS code_system,
-            {% if has_userselected %}
-            t.r.userselected
-            {% else %}
```

Comment:
Just out of curiosity, what's the compatibility issue with this?

Should you remove the python bits that set `has_userselected`?